dev_guide/secrets.adoc

```diff
@@ -116,6 +116,20 @@ Pull Secrets] for more information.
 See link:builds.html#using-private-repositories-for-builds[Using Private
 Repositories for Builds] for more information.
 
+[[service-serving-certificate-secrets]]
+=== Service Serving Certificate Secrets
+
+To secure communication to your service, you can have the cluster generate a signed
+serving certificate/key pair into a secret in your namespace.  To do this, set the
+"service.alpha.openshift.io/serving-cert-secret-name" to the name you want to use
+for your secret.  Your PodSpec can then mount that secret and when it is available
+your pod will run.  The certificate will be good for the internal service DNS name:
```

Comment:
> What is/are the file name(s) of the key/cert on the secret? File format?

PEM, like our others.  tls.crt, tls.key so that it matches the validation requirements for the tls secret type.